plugins/inventory/vmware_vm_inventory.py

```diff
@@ -97,6 +97,24 @@
                        'guest.guestId', 'guest.guestState', 'runtime.maxMemoryUsage',
                        'customValue', 'summary.runtime.powerState', 'config.guestId',
                        ]
+        subproperties:
+            description:
```

Comment:
We should document when this has been added:

```suggestion
            version_added: 4.2.0
            description:
```